offchain-modules/src/packages/config.ts

```diff
@@ -30,7 +30,19 @@ export interface EthConfig {
   privateKey: string;
 }
 
+export interface TronConfig {
+  privateKey: string;
```

Comment:
What's the difference between privateKey and committee.keys?

Reply:
privateKey deleted